src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig

```diff
@@ -1,99 +1,31 @@
-.sf-reset .traces {
-    padding: 0 0 1em 1.5em;
-}
-.sf-reset .traces a {
-    font-size: 14px;
-}
-.sf-reset .traces abbr {
-    border-bottom-color: #AAA;
-    padding-bottom: 2px;
-}
-.sf-reset .traces li {
-    color: #222;
-    font-size: 14px;
-    padding: 5px 0;
-    list-style-type: decimal;
-    margin: 0 0 0 1em;
-}
-.sf-reset .traces li.selected {
-    background: rgba(255, 255, 153, 0.5);
-}
+{{ include('@Twig/exception.css.twig') }}
```

Comment:
Why do not you use `{{ source('@Twig/exception.css') }}`?
I think that the CSS file is not supposed to be a template.